app/controllers/concerns/json_renderer.rb

```diff
@@ -57,7 +57,7 @@ def render_as_json(namespace, resource, pagy, status: :ok, allowed_includes: [])
       end
 
     # prepare resources for include collection
-    resource_json = resource.as_json(methods: requested_inlines)
+    resource_json = resource.as_json(**{methods: requested_inlines})
```

Comment:
that seems like it's exactly the same as before with more syntax ...

Reply:
Good call-out. I was troubleshooting the new way variables were passed in a few places before realizing the issue was actually with ActiveSupport. I'll clean all that up.